lib/grape/locale/en.yml

```diff
@@ -35,4 +35,10 @@ en:
         all_or_none: 'provide all or none of parameters'
         missing_group_type: 'group type is required'
         unsupported_group_type: 'group type must be Array or Hash'
+        invalid_message_body:
+          problem: "message body does not match declared format '%{body_format}'"
```

Comment:
I think this one should be without any values, so "message body does not match declared format" without the `body_format` value, then the resolution can contain that. It makes message expectations more consistent. I don't feel strongly about it though, so feel free to leave it too.